packages/nestjs-trpc/lib/generators/decorator.generator.ts

```diff
@@ -47,7 +47,7 @@ export class DecoratorGenerator {
               ...(output ? { output } : {}),
             },
           });
-        } else if (decoratorName === 'Middlewares') {
+        } else if (decoratorName === 'UseMiddlewares') {
```

Comment:
This will disable schema generation for the `Middleware` decorator.

Reply:
Should be fixed now.